src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -140,9 +140,12 @@ function getDrupalContent(buildOptions) {
         drupalData = await loadDrupal(buildOptions);
       }
       drupalData = convertDrupalFilesToLocal(drupalData, files, buildOptions);
+
       loadCachedDrupalFiles(buildOptions, files);
       pipeDrupalPagesIntoMetalsmith(drupalData, files);
       log('Successfully piped Drupal content into Metalsmith!');
+
+      metalsmith.metadata({ drupalData });
```

Comment:
By registering `drupalData` into Metalsmith metadata, that data becomes available to other plugins. I think we can use this to break up `metalsmith-drupal` into other plugins - `loadDrupalData`, `processDrupalDataIntoPages`, etc. - instead of having a single entry point doing so much.